tests/flow_comments/generics.js

```diff
@@ -0,0 +1,9 @@
+const Component = branch/*::     <Props, ExternalProps> */(
+  ({ src }) => !src,
+  // $FlowFixMe
+  renderNothing,
+)(BaseComponent);
+
+const C = b/*:: <A> */(foo) + c/*:: <B> */(bar);
+
+foo/*::<bar>*/(baz);
```

Comment:
Here’s one of the fancy new suggestions ✨

```suggestion
foo/*::<bar>*/(baz);

foo/*::<bar>*/();
```